src/main/java/org/prebid/server/auction/StoredRequestProcessor.java

```diff
@@ -134,8 +134,7 @@ public static StoredRequestProcessor create(long defaultTimeout,
     /**
      * Fetches AMP request from the source.
      */
-    public Future<BidRequest> processAmpRequest(String accountId, String ampRequestId) {
-        final BidRequest bidRequest = defaultBidRequest != null ? defaultBidRequest : BidRequest.builder().build();
```

Comment:
Where we're processing `defaultBidRequest` now?

Reply:
We will merge defaultBidRequest later in workflow. I think previous code was using defaultBidRequest to align code with auction. Currently it even more aligned
```
    private BidRequest mergeDefaultRequest(BidRequest bidRequest) {
        return jsonMerger.merge(bidRequest, defaultBidRequest, BidRequest.class);
    }
```